test/built-ins/ShadowRealm/prototype/evaluate/returns-symbol-values.js

```diff
@@ -14,10 +14,36 @@ assert.sameValue(
 );
 
 const r = new ShadowRealm();
-const s = r.evaluate('Symbol()');
+const s = r.evaluate('Symbol("foobar")');
 
 assert.sameValue(typeof s, 'symbol');
 assert.sameValue(s.constructor, Symbol, 'primitive does not expose other ShadowRealm constructor');
 assert.sameValue(Object.getPrototypeOf(s), Symbol.prototype);
-assert.sameValue(r.evaluate('Symbol.for("x")'), Symbol.for('x'));
 assert.sameValue(Symbol.prototype.toString.call(s), 'Symbol()');
```

Comment:
Wanted to double check this is correct given it gives a different result for normal symbols:
```javascript
Symbol.prototype.toString.call(Symbol("xunxo"))
> Symbol(xunxo)
```

Reply:
as I changed the value of s, I broke this assertion for not updating it. Patch coming up asap.